tests/cross/src/test/scala/tests/pc/SelectionRangeSuite.scala

```diff
@@ -179,4 +179,40 @@ class SelectionRangeSuite extends BaseSelectionRangeSuite {
     )
   )
 
+  check(
+    "def - braceless",
+    """object Main extends App :
+      |  def foo(hi: Int, b@@: Int, c:Int) = ???  """.stripMargin,
+    List(),
```

Comment:
If you want to only run for Scala 3 you can tag it on the name with `.tag(IgnoreScala2)` or similar name if I misremember